File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/ProgressMedicalReportForm.cs
namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class ProgressMedicalReportForm
    {
        public int ProgressMedicalReportFormId { get; set; }
        public int MedicalReportFormId { get; set; }
        public string NotStabilisedReason { get; set; }
        public string TreatmentDetails { get; set; }
        public string SpecialistReferralsHistory { get; set; }
        public string RadiologyFindings { get; set; }
        public string OperationsProcedures { get; set; }
        public string PhysiotherapyTreatmentDetails { get; set; }
        public System.DateTime? DateStabilised { get; set; }
        public bool? IsStabilisedChecked { get; set; }
        public bool? IsTreatmentChecked { get; set; }
        public bool? IsSpecialistReferralsHistoryChecked { get; set; }
        public bool? IsRadiologyFindingsChecked { get; set; }
        public bool? IsOperationsProceduresChecked { get; set; }
        public bool? IsPhysiotherapyTreatmentDetailsChecked { get; set; }

        public MedicalReportForm MedicalReportForm { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/digicare/src/app/medical-form-manager/models/progress-medical-report-form.ts
import { MedicalReportForm } from 'projects/digicare/src/app/medical-form-manager/models/medical-report-form'

export class ProgressMedicalReportForm {
  progressMedicalReportFormId: number;
  medicalReportForm: MedicalReportForm;
  notStabilisedReason: string;
  treatmentDetails: string;
  specialistReferralsHistory: string;
  radiologyFindings: string;
  operationsProcedures: string;
  physiotherapyTreatmentDetails: string;
  isStabilisedChecked: boolean;
  isTreatmentChecked: boolean;
  isSpecialistReferralsHistoryChecked: boolean;
  isRadiologyFindingsChecked: boolean;
  isOperationsProceduresChecked: boolean;
  isPhysiotherapyTreatmentDetailsChecked: boolean;
  dateStabilised: Date;  
}
